fusor-ember-cli/app/components/tr-hypervisor.js

```diff
@@ -29,6 +29,50 @@ export default Ember.Component.extend(SaveHostnameMixin, {
       }
       return this.send('saveHostname');
     }
-  }.observes('host.isSelectedAsHypervisor', 'custom_preprend_name', 'isCustomScheme', 'isHypervisorN', 'isFreeform', 'isMac')
+  }.observes('isSelectedAsHypervisor', 'customPreprendName', 'isCustomScheme', 'isHypervisorN', 'isFreeform', 'isMac'),
 
+  addOrRemoveHypervisor: function(){
```

Comment:
I added this from the deleted `controller/discovered_host` where I had `itemController = 'discovered-host'` in the template